pkgs/applications/editors/imhex/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, gcc10Stdenv, cmake, pkgconfig, nlohmann_json, capstone, file, glfw3, glm, jsoncpp, llvm_9, openssl, python38, fetchFromGitHub }:
+gcc10Stdenv.mkDerivation rec {
+  pname = "imhex";
+  version = "1.6.0-git"; # adds install target
+
+  src = fetchFromGitHub {
+    owner = "WerWolv";
+    repo = "ImHex";
+    #rev = "v${version}";
+    rev = "e3b5a55eba51aaab1473abf4c2a40975ddd1e9fa";
+    sha256 = "1nfq2yy21chldkg79r9nnb8n7g9fma6c9fnfy7yqgsl6y7iffxqw";
+  };
+
+  hardeningDisable = [ "format" ];
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ nlohmann_json capstone file glfw3 glm jsoncpp llvm_9 openssl python38 ];
+
+  patches = [ ./default-db.patch ];
```

Comment:
please submit the patch upstream, and fetch the corresponding commit (in your fork) with fetchpatch.